user/travis-ci-for-private.md

```diff
@@ -61,48 +60,31 @@ Travis CI is, and always will be, free for open source projects.
 For a list of plans and prices for private repositories, look at
 [travis-ci.com/plans](https://travis-ci.com/plans).
```

Comment:
For a list of plans and prices for private repositories, look at
[travis-ci.com/plans](https://travis-ci.com/plans) for not authorized users.
For a list of plans and prices for private repositories, look at
 [travis-ci.com/account/subscription](https://travis-ci.com/account/subscription) for authorized users